onnxscript/function_libs/torch_lib/ops/core.py

```diff
@@ -222,39 +222,27 @@ def aten_addcmul(
 
 @torch_op("aten::addmm")
 def aten_addmm(
+    self: TFloat, mat1: TFloat, mat2: TFloat, beta: float = 1.0, alpha: float = 1.0
+) -> TFloat:
+    """addmm(Tensor self, Tensor mat1, Tensor mat2, *, Scalar beta=1, Scalar alpha=1) -> Tensor"""
+
+    # addmm only accepts 2d tensors: https://pytorch.org/docs/stable/generated/torch.addmm.html
+    return op.Gemm(mat1, mat2, self, alpha=alpha, beta=beta)
+
+
+@torch_op("aten::addmm")
+def aten_addmm_int(
     self: TInt, mat1: TInt, mat2: TInt, beta: float = 1.0, alpha: float = 1.0
 ) -> TInt:
     """addmm(Tensor self, Tensor mat1, Tensor mat2, *, Scalar beta=1, Scalar alpha=1) -> Tensor"""
 
     mat1_mat2 = op.MatMul(mat1, mat2)
+    # NOTE: The following Mul should be optimized away by a pass when alpha and beta are 1.0
```

Comment:
Let's comment it's because onnx::gemm not supporting int type?

Reply:
That's my preference too, but I don't have strong opinion either way.

Reply:
Done